### DISCUSSION TITLE
Images mostly come out as a white screen

### DISCUSSION THREAD

**Discussion by COMANDLab** [Screenly/Anthias#1865] [General]
Hello, sorry if this has been covered before but I haven't been able to find a resolution.

I installed Anthias using a Raspberry Pi Lite with Bullseye using this guide:
https://www.hackster.io/Salmanfarisvp/setting-up-anthias-on-raspberry-pi-for-digital-signage-9f6a1b

Anthias installs fine, I can connect via browser and can upload assets.

When I create the images I want to project, however, the screen is almost always white. I tried uploading other random JPGs i have on my computer and sometimes they'd display, and sometimes they'd show as white. I've tried playing with the resolution is raspi-config as well as messing with the JPG resolution, as well as my monitor.

Does anyone know the fix for this? I imaging this is fairly trivial but Ive been spending too much time on it so I figure i'd reach out

Thanks!

**Comment by vpetersson** [comment]
I've pinged @salmanfarisvp

**Comment by salmanfarisvp** [comment]
Hi @COMANDLab , I tried to reproduce the issue on my Raspberry Pi 3B+. I used the sample image from this [site](https://file-examples.com/index.php/sample-images-download/sample-jpg-download/), it looks ok for me. 

Could you please share more details - 

- Your Raspberry Pi model
- OS version
-  SD card model and varient
- Do you have issues while using another format like PNG or JPEG or Video files? 
- Can you try an online URL like[ this](https://file-examples.com/wp-content/storage/2017/10/file_example_JPG_100kB.jpg) and share an update. 
- And finally, can you gather logs for further investigation - you can follow the guide [here](https://github.com/Screenly/Anthias/blob/master/docs/README.md)

![image](https://github.com/Screenly/Anthias/assets/11945360/48bbcbea-919a-480b-8026-2ffdfcaa0a6e)

![image](https://github.com/Screenly/Anthias/assets/11945360/574856c5-ac23-496b-929a-0ef9bcf7c955)

**Comment by COMANDLab** [comment]
Thanks for responding. I will gather some additional info shortly, but here are the answers to those i can answer now:
Thank you!!


Your Raspberry Pi model:
**4 B**
OS version

**Debian Bullseye Lite installed using the raspberry pi imager**
SD card model and varient
**Kootion 64GB Micro SD Card Ultra Micro SDXC Memory Card U3 A1 V30** 
Do you have issues while using another format like PNG or JPEG or Video files?
**Haven't tried any video files. But I tried PNG and JPEG and had the same issue. It seems variable as some large images i downloaded from the internet (like Starry Night) worked, while others didn't. Some smaller files worked, but others didn't. So its not a image loading time issue as far as I can tell. What seemed to work for me, is that if I generate an image using MS Powerpoint and use 220ppi as my resolution it seems to work**
Can you try an online URL like[ this](https://file-examples.com/wp-content/storage/2017/10/file_example_JPG_100kB.jpg) and share an update.
**Will try in a few days and get back to you**
And finally, can you gather logs for further investigation - you can follow the guide [here](https://github.com/Screenly/Anthias/blob/master/docs/README.md)
**Will try in a few days**

**Comment by salmanfarisvp** [comment]
Hi @COMANDLab, Thanks for your response. Looking forward to hearing your feedback on the image link I shared and log's for further investigation. 

Can you also share with me the image file you used for testing? Thanks.

**Comment by ckmiecik** [comment]
I am having the same issue.  I am working on a Mac so I don't think I have options to change the registry for how powerpoint saves the slides.  When I save slides from power point as images I have an option to set the resolution in pixels.  I have spent hours trying out different resolutions and found that anything over 3603x2027 makes the screen go fully white.  Anything I export that is 3603x2027 and lower works but does not fill my screen (since it is a 4k screen).  I have also tried exporting images from Adobe indesign but I still get the same white screen.  I am using a Raspberry pi4 with a 4K monitor (also same thing on a 4K Visio TV).  I used the same installation method as the original poster.

**Comment by nicomiguelino** [comment]
A PR (https://github.com/Screenly/Anthias/pull/1894) that fixes white screen issues was already merged.

- See the following comment for steps on how to update Anthias: https://github.com/Screenly/Anthias/issues/1879#issuecomment-2113971586, but make sure to backup your data first.
- Go to the **_web UI_** > **_Settings_** > **_Get Backup_**. An archive file will be downloaded; for example, `screenly-backup-2024-05-15T19-02-50.tar.gz`.
- You can also duplicate the entire micro-SD card for safety measures, before updating.
